src/internet_identity/src/stats/event_stats.rs

```diff
@@ -73,12 +74,15 @@ use std::time::Duration;
 
 /// This module defines the aggregations over the events.
 mod event_aggregations;
+
 pub use event_aggregations::*;
 
+const MAX_EVENTS_TO_PRUNE: usize = 100;
```

Comment:
This assumes that the difference between days will never be in the order of magnitude larger than 100, right?

For example, if one day we see 100.000 events, and the next day we have 10. Not all events will be pruned, right?

Reply:
Yes, if there is a huge spike in activity followed by no / very low activity then, for some time, the aggregations will be over a window that is actually longer than 24h. Eventually, the clean-up will catch up though, as for every event added, 100 are cleaned-up.